### ISSUE TITLE
Python-sklearn.preprocessing.Imputer

### ISSUE THREAD

**Issue by Zed-Wang** [Zed-Wang/ML#2]
### sklearn.preprocessing.Imputer：用于处理缺失值的插补变换器 ###
参数：
- **missing_values :**
integer or “NaN”, optional (default=”NaN”)
The placeholder for the missing values. All occurrences of missing_values will be imputed. For missing values encoded as np.nan, use the string value “NaN”.

- **strategy :** 
string, optional (default=”mean”)
The imputation strategy.
If “mean”, then replace missing values using the mean along the axis.
If “median”, then replace missing values using the median along the axis.
If “most_frequent”, then replace missing using the most frequent value along the axis.

- **axis :** 
integer, optional (default=0)
The axis along which to impute.
If axis=0, then impute along columns.
If axis=1, then impute along rows.

转载自[https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Imputer.html#sklearn.preprocessing.Imputer](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Imputer.html#sklearn.preprocessing.Imputer)
